py/server/tests/__init__.py

```diff
@@ -3,4 +3,7 @@
 #
 
 from test_helper import start_jvm
+from deephaven_server import Server
+Server().start()
+print("to call start_jvm()")
```

Comment:
I'm not sure what this println is supposed to mean.  Maybe:
```suggestion
print("Calling start_jvm()")
```